Verifying sv/ldv-linux-4.2-rc1/linux-4.2-rc1.tar.xz-43_2a-drivers--usb--host--ohci-hcd.ko-entry_point_true-unreach-call.cil.out.c
main using GenerateFunctionCallPreprocessor ...
GenerateFunctionCallPreprocessor parents for __VERIFIER_error:
__VERIFIER_error
  ldv_error
    ldv_check_alloc_nonatomic
    <end>
    ldv_check_alloc_flags_and_return_some_page
    <end>
    ldv_check_alloc_flags
      kmalloc
        fill_periodic_buffer
          debug_periodic_open
            main
            <end>
        alloc_buffer
          debug_async_open
            main
            <end>
          debug_registers_open
            main
            <end>
          debug_periodic_open
            main
            <end>
        ohci_urb_enqueue
          main
          <end>
          main
          <end>
          main
          <end>
      ldv_dma_pool_alloc_26
        ed_alloc
          ed_get
            ohci_urb_enqueue
              main
              <end>
              main
              <end>
              main
              <end>
      ldv_dma_pool_alloc_25
        td_alloc
          ed_get
            ohci_urb_enqueue
          ohci_urb_enqueue
      ldv_kmem_cache_alloc_20
      <end>
